cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/CmisSender.java

```diff
@@ -481,10 +481,10 @@ else if (getProperties) {
 		}
 	}
 
-	private Message sendMessageForActionCreate(Session cmisSession, Message message, PipeLineSession session) throws SenderException {
+	private Message sendMessageForActionCreate(Session cmisSession, Message message, PipeLineSession session, ParameterValueList pvl) throws SenderException {
 		String fileName = null;
 		try {
-			fileName = session.getMessage(getFilenameSessionKey()).asString();
+			fileName = session.getMessage( pvl.get("filenameSessionKey").asStringValue(getFilenameSessionKey()) ).asString();
```

Comment:
Ik wil graag van de filename af, hoe wordt deze nu gezet? Dit moet onderdeel zijn van 'message' en niet een los iets.

Reply:
Poeh dit was niet onderdeel van de FR 😆 

Hoe zie je dat voor je precies?

Reply:
Message heeft 'metadata' zoals de naam, size, charset, etc. die naam zou hij moeten gebruiken.

Reply:
Hmm, is er ook een manier om deze metadata vanuit de pipeline aan te passen?

We willen uiteraard regels toepassen op bestandsnamen.